packages/context/README.md

```diff
@@ -206,6 +207,24 @@ absolute URL.
 
 ##### `delete(url, options)`
 
+### Authentication Context
+
+-   [useAuthenticationContext](#useauthenticationcontext)
+
+#### `useAuthenticationContext()`
+
+Hook to obtain the authentication context based on the current logged-in user.
+
+The authentication context includes the following property and functions.
+
+-   `account` _[object]_
+    -   The account of the currently logged-in user. This includes both the user's `id` and `name`.
+-   `logout()`
+    -   A function that logs out the currently logged-in user. The user will be redirected to Evoke's logout page upon logout.
+-   `getAccessToken()`
+    -   A function that returns an access token that is associated to the currently logged-in user. This token can be used to
```

Comment:
Can you add a note that typically they should just use `ApiServices` which automatically takes care of getting a token?